great_expectations/data_context/store/store.py

```diff
@@ -194,8 +194,8 @@ def self_check(self, pretty_print: bool) -> None:
 
     def _build_key_from_config(self, config: AbstractConfig) -> DataContextKey:
         id_: Optional[str] = None
-        if hasattr(config, "id_"):
-            id_ = config.id_
+        if hasattr(config, "ge_cloud_id"):
+            id_ = config.ge_cloud_id
```

Comment:
I think this might actually be a `mercury` bug from lagging behind the `ge_cloud_id` -> `id_` transition in OSS GX. Not sure if it's easier/desirable to add this bit here temporarily or just catch up `mercury`. In any event, the call that led to this exception was `context.add_checkpoint`, which still takes `ge_cloud_id` - should prob update?